Jenkinsfile

```diff
@@ -356,21 +357,23 @@ pipeline {
             archiveArtifacts artifacts: "ee-test/container_logs/*/*", fingerprint: false, allowEmptyArchive: true
 
             publishHTML([reportDir: 'ee-test/cucumber', reportFiles: 'api/cucumber_results.html, 	authenticators_config/cucumber_results.html, \
-                                     authenticators_azure/cucumber_results.html, authenticators_ldap/cucumber_results.html, \
+                                     authenticators_ldap/cucumber_results.html, \
                                      authenticators_oidc/cucumber_results.html, authenticators_status/cucumber_results.html,\
                                      policy/cucumber_results.html , rotators/cucumber_results.html',\
                                      reportName: 'EE Integration reports', reportTitles: '', allowMissing: false, alwaysLinkToLastBuild: true, keepAll: true])
 
           }
       }
+      unstash 'testResultAzure'
```

Comment:
Probably safer to put this within a `dir` block to prevent overwriting existing files, see the unstash for `testReultEE` which is within the `ee-test` dir.

Reply:
`testReultEE`under dir because we wanted to separate the tests since those tests run twice.
so we have cucumber directory which exist to our daily runs , and when we run nightly as well.
Attached Nightly artifacts which contained both : 
![image](https://user-images.githubusercontent.com/59792097/97856800-7894ed00-1d05-11eb-8cde-8dbe12affca7.png)